SKILL.tsv

```diff
@@ -5168,7 +5168,7 @@ SKILL_20150918_005167	$Shield Guard
 SKILL_20150918_005168	$Umbo Thrust: Enhance
 SKILL_20150918_005169	$Increases the damage dealt on an enemy with [Umbo Thrust] by 1% per attribute level.
 SKILL_20150918_005170	$Moulinet: Critical
-SKILL_20150918_005171	$[Cartar Stroke] no longer knockbacks enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005171	$[Cartar Stroke] will no longer knock enemies back. (Only applies if the attribute is set to ON)
```

Comment:
To stay consistent:

will no longer knock enemies back -> no longer knocks enemies back